src/test/unit_tests/test_bucket_replication.js

```diff
@@ -0,0 +1,106 @@
+/* Copyright (C) 2020 NooBaa */
```

Comment:
You might want to add this new test to 
src/test/unit_tests/index.js

Unless you don't want it to run automatically at this point

Reply:
added, thanks.